### PR TITLE
Colors update

### PR DESCRIPTION
**Pull request by arianaescobar**
- There's no `typo0` in Hangar 
- `typo1` was using Navy Blue `#162945` instead of the correct one `#2e3c43`
- `typo2` was using Builder's `typo2` `#747d82` instead of Hangar's one `#828a8f`. 

Creating PR to review this well before merging, I don't want to break anything by removing the variable `$typo0`

### CODE REVIEW

File: src/scss/_variables.scss [hunk 1]
```diff
@@ -24,11 +24,10 @@ $docs-purple: #5300cc;
 $emerald: #2aac02;
 $base-grey: #2e3c43;
 $light-grey: rgba($base-grey, 0.02);
-$typo0: #2e3c43;
```
Comment (albertoac) [inline src/scss/_variables.scss]: You can't remove it directly as it's used in some variable (in hangar is used in the footer.scss file for the hover). Maybe is better to wait a bit before doing it, adding a deprecated comment or similar to be sure that we will not use it anymore

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -24,11 +24,10 @@ $docs-purple: #5300cc;
...
 
 // Typo Scale
-$typo1: #162945;
-$typo2: #747d82;
+$typo1: #2e3c43;
+$typo2: #828a8f;
```
Comment (albertoac) [inline src/scss/_variables.scss]: For me the new colors are ok. If those're the right ones for me is perfect!

File: src/scss/_variables.scss (hunk 1 above)
Comment (arianaescobar) [reply src/scss/_variables.scss]: oki 👍🏼

**Comment by arianaescobar** [comment]
Reviewed offline with @albertoac and decided to get rid of both `typo0` and `orange`

**Comment by arianaescobar** [comment]
@albertoac can you CR this again please? :)

**Pull request merged**